addOns/soap/src/main/java/org/zaproxy/zap/extension/soap/WSDLCustomParser.java

```diff
@@ -194,7 +194,7 @@ private void parseWSDLUrl(String url) {
                 }
             }
             /* Sends a request to retrieve remote WSDL file's content. */
-            HttpMessage httpRequest = new HttpMessage(new URI(url, false));
+            HttpMessage httpRequest = new HttpMessage(new URI(url, true));
```

Comment:
This should be noted in the changelog (and help), URLs that were working before will stop working?